s4e-web/src/app/views/map-view/map/map.component.ts

```diff
@@ -26,37 +24,37 @@ import ImageWrapper from 'ol/Image';
 })
 export class MapComponent implements OnInit, OnDestroy {
   static readonly DEFAULT_ZOOM_LEVEL = 10;
-
-  get overlays(): UIOverlay[] {
-    return this._overlays;
-  }
-
-  @Input() set overlays(value: UIOverlay[]) {
-    this._overlays = value;
-    this.overlays$.next(value);
-  }
-
-  private _overlays: UIOverlay[] = [];
-
-  private overlays$: BehaviorSubject<UIOverlay[]> = new BehaviorSubject([]);
   @Output() viewChanged = new EventEmitter<ViewPosition>();
-  private baseLayer: Layer;
-  private map: Map;
-  private activeScene$ = new ReplaySubject<Scene | null>(1);
   @ViewChild('linkDownload', {read: ElementRef}) linkDownload: ElementRef;
   @Input() isWorking: boolean = false;
   @Output() working = new EventEmitter<boolean>();
```

Comment:
Instead of having two different outputs and inputs, of `isWorking`, could you merge them as one (, because of their one functionality)? 

```
Example
@Output('isWorkingChange') emitter: EventEmitter<boolean> = new EventEmitter<boolean>();
@Input('isWorking') set isWorking(value) { ... }

```

Reply:
I do agree, but I think that it's outside of the scope of this issue. The reason is that those inputs / outputs were added at the different times.

But I'll write it down for future improvements.